src/Core/Components/DataGrid/FluentDataGridRow.razor.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.FluentUI.AspNetCore.Components;
 public partial class FluentDataGridRow<TGridItem> : FluentComponentBase, IHandleEvent, IDisposable
 {
     internal string RowId { get; set; } = string.Empty;
-    private readonly Dictionary<string, FluentDataGridCell<TGridItem>> cells = [];
+    internal Dictionary<string, FluentDataGridCell<TGridItem>> Cells { get; set; } = [];
```

Comment:
Why did you remove the `readonly`

Reply:
Had to remove it because of the getter and setter. Copy and pasted a lot of code from `FluentDataGridRow` but I will simply remove the getter and setter for the cell.

Reply:
Make sense. But in this case, the `setter` can be removed (to have a "readonly" property).